tensorzero_internal/src/endpoints/inference.rs

```diff
@@ -313,6 +314,7 @@ pub async fn inference(
                 tags: params.tags,
                 tool_config,
                 dynamic_output_schema: output_schema,
+                cached: model_used_info.cached,
```

Comment:
I don't think we return whether a non-streaming cached response was cached. We can add that if you feel strongly that we need to do this or let's just drop this.

Reply:
I thought we did, but it looks like we don't propagate `cached` all the way up to the response for non-streaming responses. I think we should have some way of indicating this, but I'll remove it for now until we come up with a unified way of doing this.